Makefile

```diff
@@ -27,8 +27,9 @@ dist: $(GENERATED)
 
 container: dist
 	cp $(OUTDIR)/ovirt-imageio-*.tar.gz container/ovirt-imageio.tar.gz
+	virt-builder fedora-36 --output container/disk.img --format qcow2
```

Comment:
Including the disk image in the container is really bad. It should be provided via --volume in the command line by the user of the container:

    podman run --volume /path/to/disk.img:/disk.img:Z ...
    
Creating the disk image each time we create a container is bad. The image should be created by the user of the container (e.g. tests) once before running the tests.

And remember that we have 2 use cases:
- download existing disk - can be a file or block device on the host
- upload to empty disk - can be a file or block device on the host